app/(dashboard)/dashboard/charts/DeploymentsByAgeChart.tsx

```
@@ -0,0 +1,91 @@
+        return acc;
+      },
+      {} as Record<string, number>
+    );
+
+    instances.forEach((instance) => {
+      if (!instance.created_at) return;
+
+      const createdAt = new Date(instance.created_at);
+      const createdMonthYear = format(startOfMonth(createdAt), "MMM yyyy");
+
+      if (counts.hasOwnProperty(createdMonthYear)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)